### ISSUE TITLE
Comments on vj in stop_times links

### ISSUE THREAD

**Issue by eturck** [hove-io/navitia#1134]
Hello,

In our process of timetable generation we, sometimes, add some comments on a vehicle_journey. It's mainly used to add an information about something we can't describe in the data.

It's working well with route_schedules but when we made some stop_schedules requests we were expecting to have those comments linked to the stop times, it was not the case.

I don't know if it's something you don't use or if you're proceeding differently. Since the vehicle_journey id is in the links, maybe you are doing another request to get the comments?

Since we were a little bit in a rush I have patched the code to have something like this : 
![stop_times](https://cloud.githubusercontent.com/assets/5136553/9137763/35d5525e-3d20-11e5-8c61-63899783d58b.png)

But before doing a PR I want to have your opinion on this.

Patch below (it's really simple) :

``` c++
diff --git a/source/type/pb_converter.cpp b/source/type/pb_converter.cpp
index 33a3dab..37c5692 100644
--- a/source/type/pb_converter.cpp
+++ b/source/type/pb_converter.cpp
@@ -1313,6 +1313,9 @@ void fill_pb_object(const navitia::type::StopTime* stop_time,
         if(!stop_time->vehicle_journey->odt_message.empty()){
             fill_pb_object(stop_time->vehicle_journey->odt_message, data, hn->add_notes(),max_depth,now,action_period);
         }
+        for (const auto& comment: data.pt_data->comments.get(*stop_time->vehicle_journey)) {
+            fill_pb_object(comment, data, hn->add_notes(), max_depth, now, action_period);
+        }
         rs_date_time->mutable_properties()->set_vehicle_journey_id(stop_time->vehicle_journey->uri);
     }
```

**Comment by TeXitoi** [comment]
Why do you need the note on each stop time? on the header is enough, no? Because the stream is already really fat, adding a link on each stop time will be very heavy.

**Comment by stifoon** [comment]
@TeXitoi : I think comments on route_schedule work well. @eturck want to add VJ comments on stop_time only on stop_schedule service.
We just have to be careful that there will not be redundant comments on route_schedule service after modifications on stop_schedule service :)

**Comment by TeXitoi** [comment]
Ok, I had in mind the route schedule data, not the stop schedule one. Forget my question.

**Comment by pbougue** [comment]
Ok, I'll have a look at that and check for possible redundancy.

**Comment by pbougue** [comment]
Closed by #1138
Thank you!
